src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorageFile.cs

```diff
@@ -7,6 +7,7 @@
 using System.Collections.Generic;
 using System.Security;
 using System.Runtime.InteropServices;
+using System.Diagnostics;
 using System.Diagnostics.Contracts;
```

Comment:
Agreed I've held back on doing a scrub of all the contract calls for now until we figure out our longer term story around them.